cupy/core/raw.pyx

```diff
@@ -24,20 +22,21 @@ cdef class RawKernel:
         options (str): Compiler options passed to NVRTC. For details, see
             https://docs.nvidia.com/cuda/nvrtc/index.html#group__options.
         backend (str): Either `nvrtc` or `nvcc`. Defaults to `nvrtc`
+        translate_cucomplex (bool): Whether the CUDA source includes the header
+            `cuComplex.h` or not. If set to ``True``, any code that uses the
+            functions from `cuComplex.h` will be translated to its Thrust
+            counterpart. Defaults to ``False``.
     """
 
-    def __init__(self, code, name, options=(), backend='nvrtc'):
-        if isinstance(code, six.binary_type):
-            code = code.decode('UTF-8')
-        if isinstance(name, six.binary_type):
-            name = name.decode('UTF-8')
-        if isinstance(backend, six.binary_type):
-            backend = backend.decode('UTF-8')
+    def __init__(self, code, name, options=(), backend='nvrtc', **kwargs):
```

Comment:
```suggestion
    def __init__(self, code, name, options=(), backend='nvrtc', *, translate_cucomplex=False):
```

Then you can remove the line below.

Reply:
Wait, I thought using `**kwargs` was what you wanted? Did I misunderstand?

Reply:
As in my original comment,

> Could you make the option a keyword-only argument so that we can easily consider reorder/renaming later?

I think [keyword-only arguments](https://python-3-for-scientists.readthedocs.io/en/latest/python3_advanced.html) (i.e., arguments list after `, * ,`) is enough. I just wanted to avoid adding new positional arguments.

Reply:
Thanks, @kmaehashi! You're right. Apparently I need some rest 😞 Will resume to address the two changes tomorrow. I'd like to see the CI outcome anyway.